apps/server/src/controllers/userControllers/getUserTransactionHistory.ts

```diff
@@ -55,7 +56,7 @@ const resolveTransactionResponse = map(
     publicTransactionFields,
     publicWalletFields,
     publicUserFields,
-    objKeysToLower
+    objKeysToCamelCase
   )
 )
 
```

Comment:
Great job Sir. Code is concise and declarative. I will come back to this when I dive deeper into Functional programming

Reply:
You don't want to make your enter gallop abi?? 😂😂😂